packages/start-server-core/tests/mocks/router-entry.ts

```
@@ -0,0 +1,30 @@
+import { AnyRouter } from '@tanstack/router-core'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)